src/altinn-app-frontend/src/features/form/layout/formLayoutSlice.ts

```diff
@@ -325,5 +324,25 @@ const formLayoutSlice = createSagaSlice(
   }),
 );
 
+const updateCommonPageSettings = (
+  state: ILayoutState,
+  page: Pick<
+    IPagesSettings,
+    'hideCloseButton' | 'showLanguageSelector' | 'showProgress' | 'triggers'
+  >,
+) => {
+  const {
+    hideCloseButton = state.uiConfig.hideCloseButton,
+    showLanguageSelector = state.uiConfig.showLanguageSelector,
+    showProgress = state.uiConfig.showProgress,
+    triggers = state.uiConfig.pageTriggers,
+  } = page;
+
+  state.uiConfig.hideCloseButton = hideCloseButton;
```

Comment:
Could this cause a condition where the global state is not set, but when going through a form it might be set by the form, and is hidden on the next process?